SpacetimeDB.ClientSDK.csproj

```
@@ -17,8 +17,8 @@
     <PackageIcon>logo.png</PackageIcon>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <RepositoryUrl>https://github.com/clockworklabs/spacetimedb-csharp-sdk</RepositoryUrl>
-    <AssemblyVersion>0.8.0</AssemblyVersion>
-    <Version>0.9.2</Version>
+    <AssemblyVersion>0.10.0</AssemblyVersion>
+    <Version>0.10.0</Version>

```

```suggestion
    <Version>$(AssemblyVersion)</Version>
```

to make future updates easier